packages/devtools_app/lib/src/debugger/flutter/codeview.dart

```diff
@@ -349,9 +347,10 @@ class GutterItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
+    final darkTheme = theme.brightness == Brightness.dark;
```

Comment:
should we make a utilityMethod `isDarkTheme(BuildContext context)` to avoid confusion about how to check for dark theme?

Reply:
sounds good! I'll look at adding an extension method on `ThemeData`, so we can do something like `Theme.of(context).isDarkTheme`.

Reply:
Added as `ThemeDataExtension`.